common/src/main/scala/org/apache/celeborn/common/util/Utils.scala

```diff
@@ -328,8 +328,7 @@ object Utils extends Logging {
         }
         isBindCollision(e.getCause)
       case e: NativeIoException =>
-        (e.getMessage != null && e.getMessage.startsWith("bind() failed: ")) ||
-          isBindCollision(e.getCause)
+        e.getMessage != null && e.getMessage.startsWith("bind")
```

Comment:
Can it pass with
```
(e.getMessage != null && e.getMessage.startsWith("bind(..) failed: ")) ||
          isBindCollision(e.getCause)
```

Reply:
The current code also looks good to me

Reply:
@turboFei, IMO, according to constructor of `NativeIoException`, `NativeIoException` has no cause and the message could only check whether message contains the `method` which is `bind`.